docs/user_guide/quick_start.rst

```diff
@@ -7,9 +7,9 @@ Build your quantum circuit
 The first step to use ``tqecd`` is to build a compatible quantum circuit.
 
 In short, compatible quantum circuits should be sequences of blocks that all follow the
-same scheme: starts by a possibly empty sequence of resets, continues by a possibly empty
+same scheme: they start with a possibly empty sequence of resets and continue by a possibly empty
```

Comment:
Repetition of `and` ("**and** continue [...] **and** end with a non-empty sequence of measurements")